CI/travis/before_install_linux

```diff
@@ -14,6 +14,12 @@ install_sphinx() {
 }
 
 handle_centos() {
+	# needed for man2html and a few other popular tools
+	yum search epel-release
```

Comment:
curios: do we need the `yum search epel-release` command?

Reply:
man2html is inside the epel-release, and no where else...